scripts/common.sh

```
@@ -43,7 +59,7 @@ function wrong_mode {
 ### Function echoes (true return) url to current user's repo (remote)
 # Return: url to repo
 function get_repo {
-    repo=$(git config --get remote.${origin_name}.url)
+    repo=$(git config --get "remote.${origin_name}.url")
```

Shellcheck (0.10.0)

[warning] origin_name is referenced but not assigned.

(SC2154)